script/openqa-clone-job

```diff
@@ -21,111 +21,114 @@
 
-  openqa-clone-job --from localhost --host localhost 42 MAKETESTSNAPSHOTS=1 FOOBAR=
+Note that the child job is the one which has the "START_AFTER_TEST" or "PARALLEL_WITH" setting
+and the parent job is the one mentioned by that setting.
 
+Cloning directly chained dependencies ("START_DIRECTLY_AFTER_TEST") is NOT supported.
 
 =head1 OPTIONS
 
 =over 4
 
 =item B<--host> HOST
 
-connect to specified host
+Specifies the hostname of the target openQA instance (defaults to localhost).
+
+Assets are still always downloaded to the local machine. When specifying a remote host
+make sure the assets are already there and use C<--skip-download>.
 
 =item B<--from> HOST
 
-get job from specified host
+Specifies the hostname of the openQA instance to clone the job from (deduced from JOBREF if it is a URL).
 
 =item B<--dir> DIR
 
-specify directory where test assets are stored (default /var/lib/openqa/factory)
+Specifies the directory to store test assets (defaults to /var/lib/openqa/factory).
 
 =item B<--skip-deps>
 
-do not clone parent jobs.
+Do NOT clone parent jobs (which is done by default).
 
 =item B<--skip-chained-deps>
 
-do not clone parent jobs of type chained. This makes the job use the downloaded hdd image instead of running the generator job again.
+Do NOT clone chained parent jobs (jobs specified via "START_AFTER_TEST").
+
+This makes the job use the downloaded HDD image instead of running the generator job again which is
+of course only possible if --host is the local machine.
 
 =item B<--skip-download>
 
-do not try any download. You need to ensure all required assets are provided yourself.
+Do NOT download assets. You need to ensure all required assets are provided yourself.
 
 =item B<--clone-children>
 
-clone child jobs as well. This is useful if you want to clone a full multi-machine cluster.
+Clone child jobs as well. This is useful if you want to clone a full multi-machine cluster.
 
 =item B<--within-instance> HOST
 
-a shortcut for C<--skip-download --from HOST --host HOST> to clone a job on a remote instance.
+A shortcut for C<--skip-download --from HOST --host HOST> to clone a job within a local or remote
+instance.
 
 =item B<--show-progress>
 
-display a progress bar of downloading asset
+Displays a progress bar when downloading assets.
 
 =item B<--parental-inheritance>
 
-provide parental job with variables from command line (they go to child job by default).
+Provides parental job with settings specified via command line (they go to child job by default).
 
 =item B<--apikey> <value>
 
-specify the public key needed for API authentication
+Specifies the public key needed for API authentication.
 
 =item B<--apisecret> <value>
 
-specify the secret key needed for API authentication
+Specifies the secret key needed for API authentication.
 
 =item B<--verbose, -v>
 
-increase verbosity
+Increases verbosity.
 
 =item B<--help, -h>
 
-print help
+Prints help.
 
 =back
 
-=head1 SYNOPSIS
```

Comment:
Do you know why there had been two SYNOPSIS sections in before? Maybe that actually makes sense in Perl POD?

Reply:
No, two sections with the same name don't really make sense

Reply:
Even if Perl people would disagree: I don't think having the same heading twice at the same level is generally considered a good way to structure a document.

Reply:
I know but there can be weird practices for good reasons :) Ok, keep it then